.github/workflows/ci-large.yml

```diff
@@ -44,6 +44,34 @@ jobs:
             hup: no_hup
             ssl: ssl
             module_type: static
+          # macOS - Wasmer (full)
+          - runtime: wasmer
+            wasmtime: 3.1.1
+            os: macos-latest
+            cc: clang
+            ngx: 1.25.3
+            ssl: ssl
+            debug: debug
+            hup: no_hup
+          # macOS - Wasmtime
+          - runtime: wasmtime
+            wasmtime: 14.0.3
+            os: macos-latest
+            cc: clang
+            ngx: 1.25.3
+            ssl: ssl
+            debug: debug
+            hup: hup
+            path: t/01-wasm
+          # macOS - V8 (full)
```

Comment:
Should we test the whole suite with V8 or is 01-wasm enough?

Reply:
Yeah, 01-wasm should be enough.